vhdbuilder/packer/vhd-image-builder-mariner-arm64.json

```diff
@@ -51,7 +56,6 @@
       },
       "location": "{{user `location`}}",
       "vm_size": "{{user `vm_size`}}",
-      "custom_data_file": "vhdbuilder/packer/sha1rsa_custom_data",
```

Comment:
do we still need this? https://github.com/Azure/AgentBaker/blob/master/vhdbuilder/packer/sha1rsa_custom_data